tests/ui/concrete-playback/array/main.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-// kani-flags: --harness harness --enable-unstable --concrete-playback=Print
+// kani-flags: --enable-unstable --concrete-playback=Print
```

Comment:
Done. I changed it to `--concrete-playback=print`.

I think we should keep it as case-insensitive since our `kani-driver` arguments seem to follow a lower-case standard while the Rust enum conventions use upper case.

Reply:
I believe our flags all follow kebab-case, but unfortunately the arguments don't. We turned case-insensitive to go around this, but ideally we should configure everything to be consistent. I couldn't find any easy fix with structopts though and we would need to migrate to using clap directly.